pkg/leaderelection/leaderelection.go

```diff
@@ -0,0 +1,54 @@
+package leaderelection
+
+import (
+	"time"
+
+	"github.com/golang/glog"
+	"github.com/k8snetworkplumbingwg/sriov-network-operator/pkg/utils"
```

Comment:
small nit: can you move this as the last import?

Reply:
absolutely, updated